app/viewmodels/journeys/adjustments/NetBusinessProfitOrLossSummary.scala

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package viewmodels.journeys.adjustments
+
+import models.journeys.adjustments.ProfitOrLoss
+import play.api.i18n.Messages
+import uk.gov.hmrc.govukfrontend.views.Aliases.Table
+import uk.gov.hmrc.govukfrontend.views.viewmodels.table.TableRow
+import viewmodels.checkAnswers.{buildTable, buildTableAmountRow}
+
+object NetBusinessProfitOrLossSummary {
+
+  private val placeholderAmount: BigDecimal = 200
```

Comment:
does placeholder means it's just  for test because we don't have a backend? I would add clearly in the comment in next PR there will come backend and we will remove this value. 

Sometimes placeholder means a text which stays and is rendered for the user inputs. 

In general, I would loud and clear comment every single test data I commit to the production code